README.md

```diff
@@ -319,6 +319,18 @@ role.
 Similar to `controller` and `vlan`, the `parent` references the connection profile in
 the ansible role.
 
+#### Network_connections Module
+
+The `network_connections.py module` is intended for internal usage by the role and It is not meant to be directly accessed or utilised by external roles or playbooks.Contains the internal Ansible module, which is the main script. It controls the communication between the role and Ansible, imports the YAML configuration and applies the changes to the provider (i.e. NetworkManager, initscripts).
```

Comment:
`Contains the internal Ansible module, which is the main script` this seems to be redundant to me, can you remove this part ?